modules/microshift-storage-vol-snapshot-class.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// microshift/volume-snapshots-microshift.adoc
+
+:_content-type: CONCEPT
+[id="microshift-volume-snapshot-classes_{context}"]
+== Volume snapshot classes
+
+Snapshotting is a CSI storage feature supported by LVMS. To enable dynamic snapshotting, at least one `VolumeSnapshotClass` must be present on the cluster.
+
+[IMPORTANT]
+====
+You must enable thin logical volumes to take logical volume snapshots.
+====
+
+.Example yaml configuration with `VolumeSnapshotClass`
+
+[source,yaml]
```

Comment:
```suggestion
.Example `VolumeSnapshotClass` <noun>
[source,yaml]
```
I think that since the `kind` is VolumeSnapshotClass, you can just use "Example VolumeSnapshotClass` Custom Resource/object/configuration file/etc